### PR TITLE
Update from regulatory-framework: 6a7a561

### PR DESCRIPTION
**Pull request by CX-Machine-User**
This PR updates the branch with the latest changes from regulatory-framework. Commit ID: 6a7a561

### CODE REVIEW

**Comment by werner-roman** [comment]
FYI - @AnnKaCatX
-> patch for jupiter

**Comment by stephanbcbauer** [comment]
Hey, the referenced files, urls must be relative something like `docs/next/`doesn't work when the version plug-in is executed. Next is in this case hardcoded and doesn't work with current or older versions.

**Comment by stephanbcbauer** [comment]
> Hey, the referenced files, urls must be relative something like `docs/next/`doesn't work when the version plug-in is executed. Next is in this case hardcoded and doesn't work with current or older versions.

Ich dödel, vergiss was ich geschrieben habe :)

File: docs/regulatory-framework/10000ft/data-exchange-contracts.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Data providers & data consumer have the option of concluding bilateral contracts
 
 ![RC Process](./assets/data-exchange-process.png)
 
-Illustration 1: RC Process with the example of an Eclipse Data Space Connector (EDC). Source: [Catena-X Operating Model](../../operating-model/why-introduction/why-introduction.md)
+Illustration 1: RC Process with the example of an Eclipse Data Space Connector (EDC). Source: [Catena-X Operating Model](/docs/next/operating-model/why-introduction/)
```
Comment (jSchuetz88) [inline docs/regulatory-framework/10000ft/data-exchange-contracts.md]:
```suggestion
Illustration 1: RC Process with the example of an Eclipse Data Space Connector (EDC). Source: [Catena-X Operating Model](./../../operating-model/why-introduction/)
```

File: docs/regulatory-framework/governance-framework.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ To easily grasp the layers of our Governance Framework, think of flight level as
...
+The ["10 Golden Rules"](/docs/next/regulatory-framework/30000ft/ten-golden-rules) document, which outlines the basic principles of the data ecosystem, is published by the Catena-X association and provides clear guidelines for necessary specifications at lower levels. It also specifies how far a technical standard can go. The agreement between the CSPs and participants (e.g., data providers and data consumers) is established during onboarding.
 
-Furthermore, the [Country Clearance List](../regulatory-framework/30000ft/country-clearance-list.md) must be observed, which shows the countries for which onboarding is possible without restrictions and the countries for which onboarding is possible after a due diligence check.
+Furthermore, the [Country Clearance List](/docs/next/regulatory-framework/30000ft/country-clearance-list) must be observed, which shows the countries for which onboarding is possible without restrictions and the countries for which onboarding is possible after a due diligence check.
```
Comment (jSchuetz88) [inline docs/regulatory-framework/governance-framework.md]:
```suggestion
Furthermore, the [Country Clearance List](./30000ft/country-clearance-list) must be observed, which shows the countries for which onboarding is possible without restrictions and the countries for which onboarding is possible after a due diligence check.
```

File: docs/regulatory-framework/governance-framework.md [hunk 3]
```diff
@@ -22,11 +22,11 @@ To easily grasp the layers of our Governance Framework, think of flight level as
...
 
-The ["10 Golden Rules"](../regulatory-framework/30000ft/ten-golden-rules.md) document, which outlines the basic principles of the data ecosystem, is published by the Catena-X association and provides clear guidelines for necessary specifications at lower levels. It also specifies how far a technical standard can go. The agreement between the CSPs and participants (e.g., data providers and data consumers) is established during onboarding.
+The ["10 Golden Rules"](/docs/next/regulatory-framework/30000ft/ten-golden-rules) document, which outlines the basic principles of the data ecosystem, is published by the Catena-X association and provides clear guidelines for necessary specifications at lower levels. It also specifies how far a technical standard can go. The agreement between the CSPs and participants (e.g., data providers and data consumers) is established during onboarding.
```
Comment (jSchuetz88) [inline docs/regulatory-framework/governance-framework.md]:
```suggestion
The ["10 Golden Rules"](./30000ft/ten-golden-rules) document, which outlines the basic principles of the data ecosystem, is published by the Catena-X association and provides clear guidelines for necessary specifications at lower levels. It also specifies how far a technical standard can go. The agreement between the CSPs and participants (e.g., data providers and data consumers) is established during onboarding.
```

File: docs/regulatory-framework/governance-framework.md [hunk 4]
```diff
@@ -22,11 +22,11 @@ To easily grasp the layers of our Governance Framework, think of flight level as
 At this level, the governance framework is defined by the Catena-X Association and the basic principles of the data ecosystem are outlined. Compliance is mandatory for all participants in the data space.
 There are several normative documents to consider.
 
-The first is the [operating model](../operating-model/why-introduction/why-introduction.md). The operating model lays the foundation and is the pre-requisite for further standardization, certification, implementation, and operations of software components and partners in the Catena-X data space. The operating model is continuously updated to reflect the evolution of business and regulatory requirements, as well as emerging technologies.
+The first is the [operating model](/docs/next/operating-model/why-introduction). The operating model lays the foundation and is the pre-requisite for further standardization, certification, implementation, and operations of software components and partners in the Catena-X data space. The operating model is continuously updated to reflect the evolution of business and regulatory requirements, as well as emerging technologies.
```
Comment (jSchuetz88) [inline docs/regulatory-framework/governance-framework.md]:
```suggestion
The first is the [operating model](./../operating-model/why-introduction). The operating model lays the foundation and is the pre-requisite for further standardization, certification, implementation, and operations of software components and partners in the Catena-X data space. The operating model is continuously updated to reflect the evolution of business and regulatory requirements, as well as emerging technologies.
```

**Review by jSchuetz88 — COMMENTED** [review]
I made some suggestions for the relative paths. I'm not 100% sure, though.

**Review by jSchuetz88 — APPROVED** [review]
LGTM.

**Pull request merged**